docs/pages/index.mdx

```diff
@@ -41,6 +52,70 @@ Started](./get-started.mdx) guide to enroll your first resource with Teleport.
 
 (!docs/pages/includes/homepage/products.mdx!)
 
+<Integrations 
+  integrations={[
+    {
+      title: 'AWS',
+      href: 'https://goteleport.com/integrations#aws',
+      iconColor: '#FF99001A',
+      iconComponent: awsSvg,
+    },
+    {
+      title: 'GCP',
+      href: 'https://goteleport.com/integrations#google-cloud-platform-(gcp)',
+      iconColor: '#EA43351A',
+      iconComponent: gcpSvg,
+    },
+    {
+      title: 'Azure',
+      href: 'https://goteleport.com/integrations#microsoft-azure',
+      iconColor: '#0078D41A',
+      iconComponent: azureSvg,
+    },
+    {
+      title: 'Okta',
+      href: 'https://goteleport.com/integrations#idp',
+      iconColor: '#0000001A',
+      iconComponent: oktaSvg,
+    },
+    {
+      title: 'Slack',
+      href: 'https://goteleport.com/integrations#plugin',
```

Comment:
https://goteleport.com/docs/identity-governance/access-request-plugins/ - technically this points them to a broader page than Slack, specifically, but this way they'll get the prompt to use the web UI before they try to follow self-hosted instructions.